frontend/src/tests/UserIdentityCard.spec.ts

```diff
@@ -0,0 +1,84 @@
+import { it, describe, beforeEach, expect} from 'vitest';
+import { mount } from '@vue/test-utils';
+import PrimeVue from 'primevue/config';
+import type { VueWrapper } from '@vue/test-utils/dist/vueWrapper';
+import type { DOMWrapper } from '@vue/test-utils/dist/domWrapper';
+import UserIdentityCard from '@/components/grantaccess/UserIdentityCard.vue';
+import { fixPrimevueCssError } from '@/tests/common/fixPrimevueCssErr';
+
+fixPrimevueCssError()
+
+describe('UserIdentityCard', () => {
+    let wrapper: VueWrapper;
+
+    let card: DOMWrapper<HTMLElement>;
+    let cardEl: HTMLDivElement;
+    let username: DOMWrapper<HTMLElement>;
+    let usernameEl: HTMLSpanElement;
+    let firstName: DOMWrapper<HTMLElement>;
+    let firstNameEl: HTMLSpanElement;
+    let lastName: DOMWrapper<HTMLElement>;
+    let lastNameEl: HTMLSpanElement;
+    let userNotExist: DOMWrapper<HTMLElement>;
+    let userNotExistEl: HTMLSpanElement;
+    let checkmarkIcon: DOMWrapper<SVGImageElement>;
+    let errorIcon: DOMWrapper<SVGImageElement>;
+
+    const props = {
+        userIdentity: {
+            userId: 'userId',
+            found: true,
+            firstName: 'First Name',
+            lastName: 'Last Name',
+        },
+    };
+
+    const propsNotFound = {
+        userIdentity: {
+            userId: 'userId',
+            found: false,
+            firstName: null,
+            lastName: null,
+        },
+    };
+
+    beforeEach(async () => {
+        wrapper = mount(UserIdentityCard, {
+            props,
+            global: {
+                plugins: [PrimeVue],
+            },
+        });
+
+        card = wrapper.find('.custom-card');
+        cardEl = card.element as HTMLInputElement;
```

Comment:
Hi Jota, do we use this one? if not could we remove it?

Reply:
Hello! I forgot to remove it. Thanks!